storage/src/main/java/com/tencent/rss/storage/util/StorageType.java

```diff
@@ -21,7 +21,7 @@
 public enum StorageType {
   HDFS,
   LOCALFILE,
-  LOCALFILE_AND_HDFS,
+  LOCALFILE_HDFS,
```

Comment:
This type is target to local file and merged file in HDFS, and in this PR, this type is target to local file and normal HDFS?
Shall we add another type for local file and merged file in HDFS?
IMO, merged file in HDFS will be removed in the feature

Reply:
1.  OK, We can give this another type for merged file in the HDFS. How about `LOCALFILE_HDFS2`?
2. IMO,  merged file in HDFS is useful for some situation. We shouldn't remove it.

Reply:
I'm ok with LOCALFILE_HDFS2
For merged file, it can be discussed in another topic